scripts/west_commands/genboard/west-ncs-genboard-test.sh

```diff
@@ -7,52 +7,29 @@ NCS_BASE="$SCRIPTDIR/../../.."
 
 SOC=${1}
 
-declare -a NCS_VERSIONS=(
-	"2.0.0"
-	"2.1.0"
-	"2.2.0"
-	"2.3.0"
-	"2.4.0"
-	"2.5.0"
-	"2.6.0"
-	"2.7.0"
-	"2.8.0"
-	"2.8.99"
-)
-
 declare -a SOCS=(
-	"nrf52805 caaa 2.0.0"
-	"nrf52810 qfaa 2.0.0"
-	"nrf52811 qfaa 2.0.0"
-	"nrf52820 qdaa 2.0.0"
-	"nrf52832 ciaa 2.0.0"
-	"nrf52832 qfaa 2.0.0"
-	"nrf52832 qfab 2.0.0"
-	"nrf52833 qdaa 2.5.0"
-	"nrf52833 qiaa 2.0.0"
-	"nrf52840 qfaa 2.5.0"
-	"nrf52840 qiaa 2.0.0"
-	"nrf5340 qkaa 2.0.0"
-	"nrf54l15 qfaa 2.7.0"
-	"nrf9131 laca 2.5.0"
-	"nrf9151 laca 2.6.0"
-	"nrf9160 sica 2.0.0"
-	"nrf9161 laca 2.5.0"
+	"nrf52805 caaa"
+	"nrf52810 qfaa"
+	"nrf52811 qfaa"
+	"nrf52820 qdaa"
+	"nrf52832 ciaa"
+	"nrf52832 qfaa"
+	"nrf52832 qfab"
+	"nrf52833 qdaa"
+	"nrf52833 qiaa"
+	"nrf52840 qfaa"
+	"nrf52840 qiaa"
+	"nrf5340 qkaa"
+	"nrf54l15 qfaa"
+	"nrf9131 laca"
+	"nrf9151 laca"
+	"nrf9160 sica"
+	"nrf9161 laca"
 )
 
 HELLO_WORLD="$NCS_BASE/../zephyr/samples/hello_world"
 
-# https://stackoverflow.com/questions/4023830
-verlte() {
-	printf '%s\n' "$1" "$2" | sort -C -V
-}
-
-verlt() {
-	! verlte "$2" "$1"
-}
-
-rm -rf $NCS_BASE/boards/arm/testvnd $NCS_BASE/boards/testvnd
-BRANCH=$(git rev-parse --abbrev-ref HEAD)
+rm -rf $NCS_BASE/boards/testvnd
```

Comment:
can we ensure NCS_BASE is **not** root, or present a warning to users before continuing? Because this is the sort of thing valve did then got a multitude of complaints when people's drives were suddenly blank

Reply:
```
SCRIPTDIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )                                                                                        
NCS_BASE="$SCRIPTDIR/../../.."   
```

hmm how could this be dangerous (just checking)?

Reply:
Assumes bash, and whilst it has `#!/bin/bash` at the top that will not guarantee it only runs in bash, and going up a directory 3 times has the possibility to up to / - see https://hackaday.com/2024/01/20/how-a-steam-bug-once-deleted-all-of-someones-user-data/

Reply:
considering this script is not for the end user (only CI), I'd really not bother.